providers/sample/src/java/org/sakaiproject/provider/user/SampleUserDirectoryProvider.java

```diff
@@ -276,7 +279,7 @@ public boolean findUserByEmail(UserEdit edit, String email)
 		if ((edit == null) || (email == null)) return false;
 
 		// assume a "@local.host"
```

Comment:
This comment should be changed/removed.
Anyway this should be merged even with this comment :smile: